api/webhook/taskcluster.go

```diff
@@ -264,9 +274,12 @@ func createAllRuns(log shared.Logger,
 		go func(browser string, urls []string) {
 			defer wg.Done()
 			log.Infof("Reports for %s: %v", browser, urls)
-			err := createRun(client, sha, api, username, password, urls, labels)
+			err := createRun(client, sha, uploadURL, username, password, urls, labels)
 			if err != nil {
 				errors <- err
+			} else {
+				browserName := strings.Split(browser, "-")[0] // chrome-dev => chrome
```

Comment:
Why do you want to extract just the browser name and leave the channel aside? We could have chrome-dev & chrome-stable on the same commit.